cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest

```diff
@@ -35,6 +35,14 @@ spec:
       name: socket
     - mountPath: /usr/bin/docker
       name: docker
+  # Add a container that runs a health-check
+  - name: nethealth-check
+    resources:
+      requests:
+        cpu: 100m
+      limits:
+        cpu: 100m
+    image:  gcr.io/google_containers/kube-nethealth-amd64:1.0
```

Comment:
Yes, this is just the seed for a more full featured nethealth monitoring container.